snapshots/financial/B3.json

```diff
@@ -0,0 +1,1260 @@
+{
```

Comment:
@arkid15r @KJhellico I don't think we can get `B3` snapshot to be generated via `make snapshot` normally on top of `BVMF` ones as it's treated as a shorter aliases at the moment instead of "alternative codes" like `ECB`/`TAR` case.

Do we wanna update the `make snapshot` to handle this or remove this manually added copy instead?

Reply:
https://github.com/vacanza/holidays/blob/a4fcbc720a5fb458ee790b7cc1c56c329937f539/holidays/registry.py#L278

Is there even a reason why it limits the alias name to between 3 and 4 characters?

Reply:
But there is no need for several snapshots for one entity. I think just one with name that matches the country/market value is enough.

Reply:
I guess I just did it because I saw the other financial markets do the same, didn't really think it through

Reply:
@KJhellico I think we should add 2-letter financial code support either way for consistency's sake though. 👀